src/witan/gwyn/utils.clj

```diff
@@ -11,3 +11,7 @@
   (if (zero? dd)
     0.0
     (/ ^double d dd)))
+
+(defn average
```

Comment:
Is this average number of pumps attending to number of fires, would be an idea to rename the function to reflect it's job? When I read it I was expecting the average as sum/count etc.